ola tests fail with a python protobuf ImportError

I cannot run the test on (an up to date) arch linux.
Seven tests are missing the module `service` from `protobuf`.

Does anyone know more about this?

<details>

<summary>Build logs</summary>

```
============================================================================
Testsuite summary for OLA 0.10.9
============================================================================
# TOTAL: 98
# PASS:  91
# SKIP:  0
# XFAIL: 0
# FAIL:  7
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log for debugging.
Some test(s) failed.  Please report this to open-lighting@googlegroups.com,
together with the test-suite.log file (gzipped) and your system
information.  Thanks.
============================================================================
make[4]: *** [Makefile:16572: test-suite.log] Error 1
make[4]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make[3]: *** [Makefile:16707: check-TESTS] Error 2
make[3]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make[2]: *** [Makefile:17594: check-am] Error 2
make[2]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make[1]: *** [Makefile:16458: check-recursive] Error 1
make[1]: Leaving directory '/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9'
make: *** [Makefile:17597: check] Error 2
==================================
   OLA 0.10.9: ./test-suite.log
==================================

# TOTAL: 98
# PASS:  91
# SKIP:  0
# XFAIL: 0
# FAIL:  7
# XPASS: 0
# ERROR: 0

System information (uname -a): Linux 6.13.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Sun, 23 Mar 2025 17:17:30 +0000 x86_64
Distribution information (/etc/os-release):
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"

.. contents:: :depth: 2

FAIL: python/ola/rpc/SimpleRpcControllerTest.sh
===============================================

Traceback (most recent call last):
  File "/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9/./python/ola/rpc/SimpleRpcControllerTest.py", line 21, in <module>
    from SimpleRpcController import SimpleRpcController
  File "/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9/python/ola/rpc/SimpleRpcController.py", line 18, in <module>
    from google.protobuf import service
ImportError: cannot import name 'service' from 'google.protobuf' (/usr/lib/python3.13/site-packages/google/protobuf/__init__.py)
FAIL python/ola/rpc/SimpleRpcControllerTest.sh (exit status: 1)

FAIL: python/ola/ClientWrapperTest.sh
=====================================

Traceback (most recent call last):
  File "/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9/./python/ola/ClientWrapperTest.py", line 26, in <module>
    from ola.ClientWrapper import ClientWrapper, _Event
  File "/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9/python/ola/ClientWrapper.py", line 29, in <module>
    from ola.OlaClient import OlaClient
  File "/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9/python/ola/OlaClient.py", line 23, in <module>
    from ola.rpc.SimpleRpcController import SimpleRpcController
  File "/home/tobiasb/.cache/rua/build/ola/src/ola-0.10.9/python/ola/rpc/SimpleRpcController.py", line 18, in <module>
    from google.protobuf import service
ImportError: cannot import name 'service' from 'google.protobuf' (/usr/lib/python3.13/site-packages/google/protobuf/__init__.py)
FAIL python/ola/ClientWrapperTest.sh (exit status: 1)

```

</details

Skipping the tests builds the package fine.
Running it also seems to works for me (I use the E1.31 and the FTDI plugin)

--- a/packages/ola/PKGBUILD
+++ b/packages/ola/PKGBUILD
@@ -61,7 +61,7 @@ package_ola() {
   depends+=(avahi libftdi-compat liblo.so libmicrohttpd.so libncursesw.so libprotobuf.so libuuid.so)
   optdepends=('python: for python bindings'
               'python-numpy: for python bindings'
-              'python-protobuf: for python bindings')
+              'python-protobuf-21: for python bindings')
   cd $pkgbase-$pkgver
   make DESTDIR="$pkgdir" install
 }

--- a/packages/ola/PKGBUILD
+++ b/packages/ola/PKGBUILD
@@ -4,14 +4,15 @@
 pkgbase=ola
 pkgname=(ola ola-docs)
 pkgver=0.10.9
-pkgrel=8
+pkgrel=9
 pkgdesc='Open Lighting Architecture for controlling entertainment lighting equipment'
 arch=(x86_64 aarch64)
 url='https://www.openlighting.org'
 license=(GPL-2.0-or-later LGPL-2.1-or-later)
 depends=()
 makedepends=(avahi cppunit doxygen flake8 libftdi-compat liblo libmicrohttpd ncurses protobuf-21
-  python-numpy python-protobuf util-linux-libs)
+  util-linux-libs)
+checkdepends=(python-numpy python-protobuf)
 source=("$pkgbase-$pkgver.tar.gz::https://github.com/OpenLightingProject/$pkgbase/archive/refs/tags/$pkgver.tar.gz"
         "fix-protobuf-v20.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1875.patch"
         "fix-liblo.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1954.patch"

--- a/packages/ola/PKGBUILD
+++ b/packages/ola/PKGBUILD
@@ -12,7 +12,7 @@ license=(GPL-2.0-or-later LGPL-2.1-or-later)
 depends=()
 makedepends=(avahi cppunit doxygen flake8 libftdi-compat liblo libmicrohttpd ncurses protobuf-21
   util-linux-libs)
-checkdepends=(python-numpy python-protobuf)
+checkdepends=(python-numpy python-protobuf-21)
 source=("$pkgbase-$pkgver.tar.gz::https://github.com/OpenLightingProject/$pkgbase/archive/refs/tags/$pkgver.tar.gz"
         "fix-protobuf-v20.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1875.patch"
         "fix-liblo.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1954.patch"

--- a/packages/ola/PKGBUILD
+++ b/packages/ola/PKGBUILD
@@ -10,8 +10,8 @@ arch=(x86_64 aarch64)
 url='https://www.openlighting.org'
 license=(GPL-2.0-or-later LGPL-2.1-or-later)
 depends=()
-makedepends=(avahi cppunit doxygen flake8 libftdi-compat liblo libmicrohttpd ncurses protobuf-21
-  util-linux-libs)
+makedepends=(avahi cppunit doxygen flake8 libftdi-compat liblo libmicrohttpd ncurses protobuf
+  protobuf-21 util-linux-libs)
 checkdepends=(python-numpy python-protobuf-21)
 source=("$pkgbase-$pkgver.tar.gz::https://github.com/OpenLightingProject/$pkgbase/archive/refs/tags/$pkgver.tar.gz"
         "fix-protobuf-v20.patch::https://github.com/OpenLightingProject/$pkgbase/pull/1875.patch"

